src/PackageDependencyViewExtension/PackageDependencyView.xaml.cs

```diff
@@ -16,6 +17,25 @@ public partial class PackageDependencyView : UserControl
 
         private WorkspaceModel currentWorkspace;
 
+        private ViewLoadedParams loadedParams;
+        private PackageDependencyViewExtension dependencyViewExtension;
+
+        private Boolean missingPackage = false;
+
+        public Boolean MissingPackage
+        {
+            get { return missingPackage; }
+            set
+            {
+                missingPackage = value;
+                if (missingPackage)
+                {
+                    loadedParams.AddToExtensionsSideBar(dependencyViewExtension, this);
```

Comment:
I think adding side effects to a setter is usually frowned upon.
I might instead raise a property change notification and implement notify property changed event - BUT that said - this class is small enough at this point it's probably okay....

I think this is a good read:
https://stackoverflow.com/questions/2908415/what-getters-and-setters-should-and-shouldnt-do

it seems to me that the side effect of opening the panel is very related to the property... so I'm okay with it, but consider the alternatives.